internals/stage_helpers.py

```diff
@@ -45,7 +45,8 @@ class StageTemplateInfo(TypedDict):
 
 def create_feature_stage(feature_id: int, feature_type: int, stage_type: int) -> Stage:
   # Create the stage.
-  stage = Stage(feature_id=feature_id, stage_type=stage_type)
+  stage = Stage(feature_id=feature_id)
```

Comment:
If it needs to be created in this specific way, then we should put a comment here otherwise a future developer could revert this change.

I am not sure that this will work because I seem to recall seeing some documentation about how datastore clients can reserve a pool of IDs and then allocate specific IDs out of that block.  But, of course, each GAE instance is a separate datastore client with its own reserved pool of IDs.  E.g., a user adds a stage using GAE instance #1 and then the same user adds another stage a few minutes later, but that POST gets processed by GAE instance #2, the IDs used could be out of order.

Instead, it might be better to add a `created` date time field to each stage.  That could also have clock skew between GAE instances but that should be less than the time between two requests from the same user.

Reply:
Changed the approach to adding a created field that is auto added, and updated when editing the stages